src/engine/source/hlp/tools/hlp_cli.cpp

```diff
@@ -0,0 +1,56 @@
+#include <stdio.h>
+#include <iostream>
+#include <iterator>
+#include <hlp/hlp.hpp>
+#include <yaml-cpp/yaml.h>
+
+int main(int argc, char * argv[])
+{
+    std::vector<std::string> logql_expressions = {};
+    std::vector<std::string> events = {};
+    if (argc < 2) {
+        std::cout << "Usage:" << std::endl;
+        std::cout << " "<< argv[0] << " \"FILENAME\"" << std::endl;
+        std::cout << " "<< argv[0] << " \"LOGQL_EXPRESSION\" \"EVENT\"" << std::endl;
+    }
+    else if (argc < 3) {
+        try {
+            YAML::Node inputs = YAML::LoadFile(argv[1]);
+            for(auto input : inputs){
+                logql_expressions.emplace_back(input["logql_expression"].as<std::string>());
+                events.emplace_back(input["event"].as<std::string>());
+            }
+        }
+        catch (const std::exception & e) {
+            std::cout << "Error reading file " << argv[1] << ".Error: " << e.what() << std::endl;
+            return 0;
+        }
+    }
+    else if (argc >= 3) {
```

Comment:
should we handle errors for more than one pair of event expression (when argc is more than 3) ?

Reply:
Now is controlled, for clarity